libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKModule.kt

```diff
@@ -157,7 +153,18 @@ class BreezSDKModule(reactContext: ReactApplicationContext) : ReactContextBaseJa
 
         try {
             val configTmp = asConfig(config) ?: run { throw SdkException.Generic("Missing mandatory field config of type Config") }
+
+            if (!configTmp.workingDir.contains(reactApplicationContext.filesDir.toString())) {
```

Comment:
Not sure if we should limit here the directory path.

Reply:
I understand we shouldn't limit too much, the application only has access to the application sandbox directory, documents directory and external file system. Throwing the above exception with a similar error message, would also suffice

Reply:
Updated to not limit the workingDir and provide a better error if creation of the workingDir fails